<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>ADFG(V)X cipher</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Text for encryption:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageInput">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyInput">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>621</width>
      <height>211</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>710</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypted message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="encResult">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>710</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="language_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Language: Default (English)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="type_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of cipher: Default (ADFGX)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="enc_dec_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>650</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt / Decipher</string>
    </property>
   </widget>
   <widget class="QLabel" name="warnings">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="fillingTable_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filling table: Default (Randomly generated)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Warnings:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_encrypt">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_decipher">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>600</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Decipher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remaining characters to put in table:</string>
    </property>
   </widget>
   <widget class="QLabel" name="remaining">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>751</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>780</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Deciphered message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="decResult">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>780</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Type</string>
    </property>
    <addaction name="ADFGX"/>
    <addaction name="ADFGVX"/>
   </widget>
   <widget class="QMenu" name="menuLanguage_2">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="ENG"/>
    <addaction name="CZE"/>
   </widget>
   <widget class="QMenu" name="menuFilling_table">
    <property name="title">
     <string>Filling table</string>
    </property>
    <addaction name="generate"/>
    <addaction name="user"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuLanguage_2"/>
   <addaction name="menuFilling_table"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ADFGX">
   <property name="text">
    <string>ADFGX</string>
   </property>
  </action>
  <action name="ADFGVX">
   <property name="text">
    <string>ADFGVX</string>
   </property>
  </action>
  <action name="ENG">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="CZE">
   <property name="text">
    <string>Czech</string>
   </property>
  </action>
  <action name="generate">
   <property name="text">
    <string>Randomly generate</string>
   </property>
  </action>
  <action name="user">
   <property name="text">
    <string>By user</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>